docs/installation/central-collector.md

```diff
@@ -1,3 +1,236 @@
+    grid certificate for your Central Collector's host certificate:
+
+    1.  Set the following configuration in `/etc/condor-ce/config.d/01-ce-auth.conf`:
+
+            AUTH_SSL_SERVER_CERTFILE = /etc/grid-security/hostcert.pem
+            AUTH_SSL_SERVER_KEYFILE = /etc/grid-security/hostkey.pem
+            AUTH_SSL_SERVER_CADIR = /etc/grid-security/certificates
+            AUTH_SSL_SERVER_CAFILE =
+            AUTH_SSL_CLIENT_CERTFILE = /etc/grid-security/hostcert.pem
+            AUTH_SSL_CLIENT_KEYFILE = /etc/grid-security/hostkey.pem
+            AUTH_SSL_CLIENT_CADIR = /etc/grid-security/certificates
+            AUTH_SSL_CLIENT_CAFILE =
+
+    1.  Install your host certificate and key into `/etc/grid-security/hostcert.pem` and `/etc/grid-security/hostkey.pem`,
+        respectively
+
+    1.  Set the ownership and Unix permissions of the host certificate and key
+
+            :::console
+            root@host # chown root:root /etc/grid-security/hostcert.pem /etc/grid-security/hostkey.pem
+            root@host # chmod 644 /etc/grid-security/hostcert.pem
+            root@host # chmod 600 /etc/grid-security/hostkey.pem
+
+-  Otherwise, use the default system locations:
+
+    1.  Install your host certificate and key into `/etc/pki/tls/certs/localhost.crt` and
+        `/etc/pki/tls/private/localhost.key`, respectively
+
+    1.  Set the ownership and Unix permissions of the host certificate and key
+
+            :::console
+            root@host # chown root:root /etc/pki/tls/certs/localhost.crt /etc/pki/tls/private/localhost.key
+            root@host # chmod 644 /etc/pki/tls/certs/localhost.crt
+            root@host # chmod 600 /etc/pki/tls/private/localhost.key
+
+### Optional configuration ###
+
+The following configuration steps are optional and will not be required for all Central Collectors.
+If you do not need any of the following special configurations, skip to
+[the section on next steps](#distributing-configuration-to-site-htcondor-ces).
+
+#### Banning HTCondor-CEs ####
+
+By default, Central Collectors accept ClassAds from all HTCondor-CEs with a valid and accepted certificate.
+If you want to stop accepting ClassAds from a particular HTCondor-CE, add its hostname to `DENY_ADVERTISE_SCHEDD` in
+`/etc/condor-ce/config.d/01-ce-collector.conf`.
+For example:
+
+```
+DENY_ADVERTISE_SCHEDD = $(DENY_ADVERTISE_SCHEDD), misbehaving-ce-1.bad-domain.com, misbehaving-ce-2.bad-domain.com
+```
+
+#### Configuring HTCondor-CE View ####
+
+The HTCondor-CE View is an optional web interface to the status of your CE.
+To run the HTCondor-CE View, install the appropriate package and set the relevant configuration.
+
+1.  Begin by installing the `htcondor-ce-view` package:
+
+        :::console
+        root@host # yum install htcondor-ce-view
+
+2.  Next, uncomment the `DAEMON_LIST` configuration located at `/etc/condor-ce/config.d/05-ce-view.conf`:
+
+        DAEMON_LIST = $(DAEMON_LIST), CEVIEW, GANGLIAD, SCHEDD
```

Comment:
In v4 it's already uncommented.

Reply:
at least for OSG. Let me check if that's true for UW as well.

Reply:
Yeah it's in https://github.com/htcondor/htcondor-ce/blob/master/config/05-ce-view-defaults.conf